lib/dor/services/client/object.rb

```diff
@@ -65,6 +65,25 @@ def find
           raise_exception_based_on_response!(resp)
         end
 
+        # Updates the object
+        # @param [Cocina::Models::RequestDRO,Cocina::Models::RequestCollection,Cocina::Models::RequestAPO]
```

Comment:
I think you're missing `params` here between `@param` and the list of types

Reply:
Tweaked. Placed after to be consistent with other YARD docs.